src/components/Tree/Tree.js

```diff
@@ -132,9 +135,21 @@ export const Tree = ({
     updateModel()
   }
 
+  /**
+   * Sets the filterText state
+   * @param {Object} e event object
+   */
+  const onFilterChange = (e) => {
+    setFilterText(e.target.value.toLowerCase())
```

Comment:
May want to trim this as well while you're formatting -- how leading/trailing whitespace function?

Reply:
I just tried trimming it here and that doesn't let you add a space at all. I added a trim when passing it to TreeItem and that behaves better